Library/Homebrew/language/node.rb

```diff
@@ -16,6 +16,12 @@ def self.pack_for_installation
       # fed to `npm install` only symlinks are created linking back to that
       # directory, consequently breaking that assumption. We require a tarball
       # because npm install creates a "real" installation when fed a tarball.
+      if File.exist?("package.json")
+        pkg_json = JSON.parse(IO.read("package.json"))
+        pkg_json["scripts"]&.delete("prepare")
+        pkg_json["scripts"]&.delete("prepack")
+        IO.write("package.json", JSON.pretty_generate(pkg_json))
```

Comment:
```suggestion
        pkg_json["scripts"]&.delete("prepare")
        pkg_json["scripts"]&.delete("prepack")
        IO.write("package.json", JSON.pretty_generate(pkg_json))
```

If `pkg_json["scripts"]` doesn't exist: is that a problem or a no-op? If the prior: let's `raise`. If the latter: let's avoid writing out the same file.

Reply:
I think it should be a no-op. I've updated it to only re-write the package.json if something was removed.